src/app/components/nav/nav.component.ts

```diff
@@ -34,4 +34,23 @@ export class NavComponent implements OnInit {
     $('#navModalContentUpload').modal('toggle');
   }
 
+  isAuthenticated() {
+      return this.auth.isAuthenticated();
+  }
+
+  isAllowed() {
```

Comment:
`isAllowed` or `isManager`? Choose one. If you choose the first one, please change to "is allowed to <whatever operation>".

Reply:
I didn't wanted to give clues about the logic so I have choosed isAllowedTo.